### PR TITLE
OTP integration:fix merge issue on Document Processor

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

**Review by toggm — CHANGES_REQUESTED** [review]
There are other locations where `parseFormat` is still used wrongly. You need to replace all of them.

**Comment by mcmpp** [comment]
Sorry, I didn't see it. I have submitted now all functions and modifications on the main function

File: src/main/scala/ch/openolitor/core/reporting/DocumentProcessor.scala [hunk 1]
```diff
@@ -681,10 +659,14 @@ trait DocumentProcessor extends LazyLogging {
...
-      name.split('|').toList match {
-        case name :: Nil  => (correctPropertyName(name.trim), Nil)
-        case name :: tail => (correctPropertyName(name.trim), tail.map(_.trim))
-        case _            => (correctPropertyName(name), Nil)
+      val propertyName = name match {
+        case propertyPattern(frameName) => frameName
+        case fullName                   => fullName
+      }
+      propertyName.split('|').toList match {
+        case name :: Nil  => (correctPropertyName(propertyName.trim), Nil)
```
Comment (toggm) [inline src/main/scala/ch/openolitor/core/reporting/DocumentProcessor.scala]: actual prod has:
```
propertyName.split('|').toList match {
        case name :: Nil  => (correctPropertyName(name.trim), Nil)
        case name :: tail => (correctPropertyName(name.trim), tail.map(_.trim))
        case _            => (correctPropertyName(propertyName), Nil)
      }
```

Comment (mcmpp) [reply src/main/scala/ch/openolitor/core/reporting/DocumentProcessor.scala]: done

**Review by toggm — APPROVED** [review]

**Pull request merged**